layouts/default.vue

```diff
@@ -40,15 +40,13 @@
 import { Component, Vue } from 'nuxt-property-decorator'
 import ScaleLoader from 'vue-spinner/src/ScaleLoader.vue'
 
+import { NuxtConfig, NuxtOptionsHead as MetaInfo } from '@nuxt/types'
```

Comment:
```suggestion
import type { NuxtOptionsHead as MetaInfo } from '@nuxt/types/config/head'
import type { NuxtConfig } from '@nuxt/types'
import type { LinkPropertyHref } from 'vue-meta'
```
`NuxtOptionsHead` は `@nuxt/types` には無さそうなので別途引っ張ってくる必要がありそうです。  `LinkPropertyHref` についても `vue-meta` のを使いたい（より定義が細かいので）